cmd/kubeadm/app/util/staticpod/volumes.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package staticpod
+
+import "k8s.io/api/core/v1"
+
+// NewVolume creates a v1.Volume with a hostPath mount to the specified location
+func NewVolume(name, path string) v1.Volume {
```

Comment:
nit: in order to change a little bit less lines, could you just have one file in this dir called `utils.go` that has all these funcs? I think it's so little code it can fit into one file.